src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowEurostagTutorialExample1Test.java

```diff
@@ -409,4 +409,26 @@ void testWithDisconnectedGenerator() {
         assertActivePowerEquals(Double.NaN, gen.getTerminal());
         assertReactivePowerEquals(Double.NaN, gen.getTerminal());
     }
+
+    @Test
+    void testGeneratorReactiveLimits() {
+        Network network = EurostagTutorialExample1Factory.create();
+        network.getGenerator("GEN").newMinMaxReactiveLimits().setMinQ(0).setMaxQ(120).add();
+        network.getVoltageLevel("VLGEN").newGenerator().setId("GEN1")
+                .setBus("NGEN").setConnectableBus("NGEN")
+                .setMinP(-9999.99D).setMaxP(9999.99D)
+                .setVoltageRegulatorOn(true).setTargetV(24.5D)
+                .setTargetP(607.0D).setTargetQ(301.0D).add();
+        network.getGenerator("GEN1").newMinMaxReactiveLimits().setMinQ(0).setMaxQ(220).add();
+        LoadFlowParameters parameters = new LoadFlowParameters().setNoGeneratorReactiveLimits(false)
+                .setDistributedSlack(false)
+                .setVoltageInitMode(LoadFlowParameters.VoltageInitMode.DC_VALUES);
+        loadFlowRunner.run(network, parameters);
+        network.getGenerators().forEach(gen -> {
```

Comment:
It would be better to have simple assertions with hard coded value, for example checking that generator GEN1 is at its maximum limit (120) and GEN2 is (or is not I don't know :) ).